spec/integration/execution_result_spec.rb

```diff
@@ -3,21 +3,25 @@
 require 'bolt_spec/integration'
 require 'bolt/cli'
 
-describe "when runnning over the ssh transport", ssh: true do
+describe "when running a plan that manipulates an execution result", ssh: true do
   include BoltSpec::Conn
   include BoltSpec::Files
   include BoltSpec::Integration
 
   let(:modulepath) { File.join(__dir__, '../fixtures/modules') }
-  let(:uri) { conn_uri('ssh') }
-  let(:user) { conn_info('ssh')[:user] }
-  let(:password) { conn_info('ssh')[:password] }
+  let(:uri) { conn_uri('ssh', true) }
 
   after(:each) { Puppet.settings.send(:clear_everything_for_tests) }
 
   context 'when using CLI options' do
     let(:config_flags) { %W[--insecure --format json --modulepath #{modulepath}] }
 
+    it 'returns true on success' do
```

Comment:
These integration tests could use something to suppress the logging output. With the change to log notices from a plan by default, we now get them showing up during test runs.